File: account-service/model/user_account.py
class UserAccount:
    def __init__(self, user_account_id: int, account_id: int, user_id: int, account_type_id: int, account_name: str, icon: str, account_type_1: str, account_type_2: str, account_type_3: str, balance: float, balance_date: str):
        self.user_account_id: int = user_account_id
        self.account_id: int = account_id
        self.user_id: int = user_id
        self.account_type_id: int = account_type_id
        self.account_name: str = account_name
        self.icon: str = icon
        self.account_type_1: str = account_type_1
        self.account_type_2: str = account_type_2
        self.account_type_3: str = account_type_3
        self.latest_balance: float = balance
        self.latest_balance_date: str = balance_date

File: account-service/src/main/java/com/finance/manager/account_service/dto/UserAccount.java
package com.finance.manager.account_service.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Builder;

import java.math.BigDecimal;

@Builder
public record UserAccount(
        @JsonProperty("user_account_id")
        int userAccountId,

        @JsonProperty("account_id")
        int accountId,

        @JsonProperty("user_id")
        int userId,

        @JsonProperty("user_account_name")
        String userAccountName,

        @JsonProperty("account_type_id")
        int accountTypeId,

        @JsonProperty("account_name")
        String accountName,

        @JsonProperty("statement_file_extensions")
        String statementFileExtensions,

        @JsonProperty("icon")
        String icon,

        @JsonProperty("account_type_1")
        String accountType1,

        @JsonProperty("account_type_2")
        String accountType2,

        @JsonProperty("account_type_3")
        String accountType3,

        @JsonProperty("latest_balance")
        BigDecimal latestBalance,

        @JsonProperty("latest_balance_date")
        String latestBalanceDate
) {

}